### ISSUE TITLE
Add the ability to spawn additional threads after mioco.start

### ISSUE THREAD

**Issue by Drakulix** [dpc/mioco#74]
Currently the thread pool is fixed in size after starting mioco.
It would be nice for scheduling purposes to be able to add (and maybe remove) threads to the pool.

**Comment by dpc** [comment]
Agreed. But this non-essential feature, so I'd leave that our for a while until other things are settled and agreed on.
